package.json

```diff
@@ -52,13 +52,13 @@
     "microbundle": "0.12.4",
     "prettier": "2.1.2",
     "raf": "3.4.1",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
+    "react": "17.0.1",
```

Comment:
devDependencies should probably stay on 16 until enzyme supports 17, otherwise the tests won't pass

Reply:
Tests will pass with 17 as well, but agreed that waiting for 17 adapter would be nice.

Reply:
So let's maybe update only the peer dependencies for now?

Reply:
Yes please

Reply:
peerDependencies updated in #119